doi-transformer/src/main/java/no/unit/nva/doi/transformer/CrossRefConverter.java

```diff
@@ -358,7 +373,7 @@ private BookMonograph createBookMonograph(CrossRefDocument document) {
 
     private ChapterArticle createChapterArticle(CrossRefDocument document) {
         return new ChapterArticle.Builder()
-                .withPages(extractPages(document))
+                .withPages((Range) extractPages(document))
```

Comment:
StringUtils.parsePage returns Range (indicating it should be called parseRange…) so the cast shouldn't be necessary.

Maybe indicates too that the method parseMonographPages should be in StringUtils…or that StringUtils is the place where Pages is retuned and everything is checked with instance of instead…not sure what. the best approach is